src/common/monitors.cpp

```diff
@@ -1,84 +1,99 @@
 #include "pch.h"
 
-#include "common.h"
 #include "monitors.h"
 
+#include <Shellscalingapi.h>
+
+#include "common.h"
+
 bool operator==(const ScreenSize& lhs, const ScreenSize& rhs)
 {
     auto lhs_tuple = std::make_tuple(lhs.rect.left, lhs.rect.right, lhs.rect.top, lhs.rect.bottom);
     auto rhs_tuple = std::make_tuple(rhs.rect.left, rhs.rect.right, rhs.rect.top, rhs.rect.bottom);
     return lhs_tuple == rhs_tuple;
 }
 
-static BOOL CALLBACK get_displays_enum_cb(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
+static BOOL CALLBACK getDisplaysEnumCb(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
 {
-    MONITORINFOEX monitor_info;
-    monitor_info.cbSize = sizeof(MONITORINFOEX);
-    GetMonitorInfo(monitor, &monitor_info);
-    reinterpret_cast<std::vector<MonitorInfo>*>(data)->emplace_back(monitor, monitor_info.rcWork);
+    MONITORINFOEX monitorInfo;
+    monitorInfo.cbSize = sizeof(MONITORINFOEX);
+    if (GetMonitorInfo(monitor, &monitorInfo))
+    {
+        reinterpret_cast<std::vector<MonitorInfo>*>(data)->emplace_back(monitor, monitorInfo.rcWork);
+    }
     return true;
 };
 
-static BOOL CALLBACK get_displays_enum_cb_with_toolbar(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
+static BOOL CALLBACK getDisplaysEnumCbWithToolbar(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
 {
-    MONITORINFOEX monitor_info;
-    monitor_info.cbSize = sizeof(MONITORINFOEX);
-    GetMonitorInfo(monitor, &monitor_info);
-    reinterpret_cast<std::vector<MonitorInfo>*>(data)->emplace_back(monitor, monitor_info.rcMonitor);
+    MONITORINFOEX monitorInfo;
+    monitorInfo.cbSize = sizeof(MONITORINFOEX);
+    if (GetMonitorInfo(monitor, &monitorInfo))
+    {
+        reinterpret_cast<std::vector<MonitorInfo>*>(data)->emplace_back(monitor, monitorInfo.rcMonitor);
+    }
     return true;
 };
 
 std::vector<MonitorInfo> MonitorInfo::GetMonitors(bool include_toolbar)
 {
     std::vector<MonitorInfo> monitors;
-    EnumDisplayMonitors(NULL, NULL, include_toolbar ? get_displays_enum_cb_with_toolbar : get_displays_enum_cb, reinterpret_cast<LPARAM>(&monitors));
+    EnumDisplayMonitors(NULL, NULL, include_toolbar ? getDisplaysEnumCbWithToolbar : getDisplaysEnumCb, reinterpret_cast<LPARAM>(&monitors));
     std::sort(begin(monitors), end(monitors), [](const MonitorInfo& lhs, const MonitorInfo& rhs) {
         return lhs.rect < rhs.rect;
     });
     return monitors;
 }
 
-int MonitorInfo::GetMonitorsCount()
+bool MonitorInfo::DoesAllMonitorsHaveSameDpiScaling()
```

Comment:
`DoesAllMonitorsHaveSameDpiScaling` -> `AllMonitorsHaveSameDpiScaling`

Reply:
I think if function is predicate it name should be question as its more clean , but ok

Reply:
Better to keep the verbosity to a certain level, the fact that the function returns a boolean makes it pretty clear that it does.